infrastructure/oss/cli_scanner.go

```diff
@@ -263,10 +263,15 @@ func (cliScanner *CLIScanner) unmarshallAndRetrieveAnalysis(ctx context.Context,
 	}
 
 	for _, scanResult := range scanResults {
-		targetFilePath := path
 		targetFile := cliScanner.determineTargetFile(scanResult.DisplayTargetFile)
-		if targetFile != "" {
+		targetFilePath := scanResult.Path
+		if targetFilePath == "" {
+			targetFilePath = workDir
+		}
+		if targetFilePath == "" {
 			targetFilePath = filepath.Join(workDir, targetFile)
```

Comment:
if targetFilePath is still empty here that means workDir is also empty. So no need to do filepath.Join?